api/src/test/java/com/ovh/mls/serving/runtime/huggingface/HuggingFaceTokenizerTest.java

```diff
@@ -0,0 +1,67 @@
+package com.ovh.mls.serving.runtime.huggingface;
+
+import com.jayway.restassured.RestAssured;
+import com.jayway.restassured.response.Header;
+import com.ovh.mls.serving.runtime.core.ApiServer;
+import com.typesafe.config.Config;
+import com.typesafe.config.ConfigFactory;
+import io.swagger.v3.oas.integration.OpenApiContextLocator;
+import org.hamcrest.core.IsEqual;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+import java.lang.reflect.Field;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import static com.jayway.restassured.RestAssured.given;
+import static org.hamcrest.Matchers.equalTo;
+
+public class HuggingFaceTokenizerTest {
+
+    @BeforeAll
+    public static void startServer() throws NoSuchFieldException, IllegalAccessException {
+        // Reset swagger context
```

Comment:
Why is it necessary to reset the swagger ?

Reply:
I'm not sure I copy-pasted it from an other unit test. Also the Api port is always different. @XciD do you know why ?

Reply:
You need to reset the swagger in order to re-gen it. Don't needed on this case I think.
And about the port, you are right, you need a random one if tests runs in parallel